rosbag2_examples/rosbag2_examples_py/rosbag2_examples_py/simple_bag_player.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+
+import rclpy
+from rclpy.node import Node
+from rclpy.serialization import deserialize_message
+import rosbag2_py
+from std_msgs.msg import String
+
+
+class SimpleBagPlayer(Node):
+
+    def __init__(self, bag_filename):
+        super().__init__('simple_bag_player')
+        self.reader = rosbag2_py.SequentialReader()
+        storage_options = rosbag2_py._storage.StorageOptions(
+            uri=bag_filename,
+            storage_id='sqlite3')
```

Comment:
if i am not mistaken, this is intentionally enabled as `sqlite3`? but this bag file can be also red from `simple_bag_player.cpp`, because it reads the meta data from the bag file and use the appropriate storage plugin if available? just checking this because user does not meet the failure during the tutorial.

Reply:
It was set as `sqlite3` in `simple_bag_recorder.py`. I don't know the reason exactly.
It is possible to read `sqlite3` files from `simple_bag_player.cpp`, but the opposite(reading `mcap` files from `simple_bag_player.py`) isn't.

Maybe we can delete `storage_id`?

Reply:
No, i think that is okay with current code. this also includes an example how to specify the storage id, i think that is fine.